projects/moment/package.json

```
@@ -5,11 +5,11 @@
   "author": "PL <kuhnroyal@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/cdk": "^20.0.0",
-    "@angular/common": "^20.0.0",
-    "@angular/core": "^20.0.0",
-    "@angular/material": "^20.0.0",
-    "@angular/material-moment-adapter": "^20.0.0",
+    "@angular/cdk": "^21.0.0",
+    "@angular/common": "^21.0.0",
+    "@angular/core": "^21.0.0",
+    "@angular/material": "^21.0.0",
+    "@angular/material-moment-adapter": "^21.0.0",
     "@mat-datetimepicker/core": "^16.0.0"
```

```suggestion
    "@mat-datetimepicker/core": "^17.0.0"
```

sorry, by ["leave be"](https://github.com/kuhnroyal/mat-datetimepicker/pull/672#discussion_r2553951110) I meant the the dependency should stay on v17. this one is _not_ automatically bumped (because it's a range). I can fix that when I check the lockfile.